docs/source/tutorial_model_cards.md

```diff
@@ -0,0 +1,181 @@
+   
+   Currently, there are two different modes `editing` or `final` for step 2.
+   For the `editing` mode, the code will generate messages like  this:
+
+   > :warning: missing *section name*: Probably need to be grabbed from paper & added to model_list.py by u (the creator) :warning:
+
+   In `final` mode, such messages will not exist. By default, the `mode` is `editing`. 
+
+
+
+### Using `extra-args-path`
+
+We can use `extra-args-path` to pass in longer arguments. By default, `extra-args-path` will be `<folder-to-save>/args.json`. 
+
+#### Adding Custom Dataset and Model Info
+By default, the code will try to find a sections in [`model_list.py`](https://github.com/facebookresearch/ParlAI/blob/master/parlai/zoo/model_list.py). However, instead of changing `model_list.py`, we can also pass in a `.json` file to `--extra-args-path` with out new section. Here's us trying to add the intended use section
+
+```
+# args.json
+{
+   "extra_models": {
+      "zoo:blender/blender_90M/model": {
+         # section name (lowercased and underscores removed): section content
+         "privacy": "Our model is intended for research purposes only, and is not yet production ready...."
+      }
+   }   
+}
+```
+
+Similarly, if we don't want to touch [`task_list.py`](https://github.com/facebookresearch/ParlAI/blob/master/parlai/tasks/task_list.py) (information about the tasks), we can also pass the details via `--extra-args-path`. Here's us trying add a description for `dummy_task`:
+```
+# args.json
+{
+   "extra_tasks": {
+      "dummy_task": {
+         # type of info: info
+         "description": "This is a dummy task, not a real task"
+      }
+   }   
+}
+```
+The information passed via this method can partially overwrite what's written in `task_list.py` and `model_list.py`. 
+
+
+#### Add Custom Sections or Changing Section Order (static)
+
+For static sections, there's two ways to do this. 
+
+1. After we generate the inital model card, we can directly edit the generated markdown file.
+
+3. If there's a lot section movement or deletion, use add a `user_sections` key to specify the entire section order to the `.json` file that we pass to `--extra-args-path`. For instance, this is the default order of sections: 
+    ```
+         section_list = [
+            "model_details",
+            "model_details:_quick_usage",
+            "model_details:_sample_input_and_output",
+            "intended_use",
+            "limitations",
+            "privacy",
+            "datasets_used",
+            "evaluation",
+            "extra_analysis",
+            "related_paper",
+            "hyperparameters",
+            "feedback",
+         ]
+    ```
+   Note that adding `:_` implies that it's a subsection.
+
+   Here's us trying to to reverse the order and remove the model_details section (for kudos):
+   ```
+   # args.json
+   {
+      "user_section_list": [
```

Comment:
up above you say that the key is `user_sections`, not `user_section_list`?

Reply:
oops